spec/fixtures/custom_plugins/kong/plugins/short-circuit-last/schema.lua

```diff
@@ -4,9 +4,6 @@ local typedefs = require "kong.db.schema.typedefs"
 return {
   name = "short-circuit-last",
   fields = {
```

Comment:
@dndx ah, I think those fixtures for "short-circuit" and "short-circuit-last" exist only for the plugins-triggering tests that were removed by this PR, and can be removed as well.

Reply:
Hmm, I think those plugins are still used for `init_worker` tests:

https://github.com/Kong/kong/blob/8ee57d885756322bc89e31fa997132d74491beee/spec/02-integration/05-proxy/04-plugins_triggering_spec.lua#L2388

Can we safely remove those tests too?

Reply:
ah, didn't notice `short-circuit` was doing double-duty for that as well! at the very least `short-circuit-last` could go away, I guess?

> Can we safely remove those tests too?

No, I think these tests still apply.